.eslintrc.json

```diff
@@ -12,8 +12,6 @@
     "plugin:react/recommended",
     "plugin:jsx-a11y/recommended",
     "plugin:react-hooks/recommended",
-    "plugin:jest/recommended",
-    "plugin:testing-library/react"
```

Comment:
Removing this line _does_ help reduce the warnings in the test files, but notice that doing so leaves a trailing comma (line 14), which isn't permitted in JSON.